src/cluster_legacy.c

```diff
@@ -4938,7 +4938,9 @@ void clusterPropagatePublish(robj *channel, robj *message, int sharded) {
     clusterNode *node;
     while ((node = clusterNodeIterNext(&iter)) != NULL) {
         if (node->flags & (CLUSTER_NODE_MYSELF | CLUSTER_NODE_HANDSHAKE)) continue;
+        if (!node->link || node->pong_received < node->link->ctime) continue;
         if (nodeSupportsLightMsgHdrForPubSub(node)) {
+            if (!node->link || node->pong_received < node->link->ctime) continue;
```

Comment:
Maybe add some comments for clarity.
```suggestion
            /* Avoid sending pub/sub light weight message until the bidirectional link(s) have been established. */
            if (!node->link || node->pong_received < node->link->ctime) continue;
```